src/hacbs/components/PipelineRunDetailsView/TriggeredBySection.scss

```diff
@@ -0,0 +1,7 @@
+.odc-pipeline-run-details {
```

Comment:
Should we have `hacbs-` prefix for all the component styles that are defined inside hacbs folder ?

Reply:
The content of this file can be moved to `PipelineRunDetails.scss` and this file can be removed, wdyt?

Reply:
this style wasn't being used yet so i remove it, but applied a hacbs prefix to the two styles i was using - thank you